.github/workflows/ci.yml

```diff
@@ -79,6 +79,26 @@ jobs:
         target:
           - x86_64-apple-darwin
 
+  cc:
+    runs-on: ubuntu-22.04
+    env:
+      TARGET_TRIPLE: x86_64-unknown-linux-gnu
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          submodules: true
+      - uses: dtolnay/rust-toolchain@stable
+      # Ensures --all-features builds correctly, the current logic will mean it
+      # uses stock zlib, not cmake nor cc
+      - run: |
+          cargo test --target $TARGET_TRIPLE --all-features
+          cargo run --target $TARGET_TRIPLE --manifest-path systest/Cargo.toml --all-features
+      # Ensures zlib-ng builds and runs, though zlib-ng _could_ change internally
+      # and not use all optimizations available to the CI runner
+      - run: |
```

Comment:
This run block seems to test more zlib-ng-no-cmake features which might fail, even though this job should only fail if `--all-features` isn't working as expected (i.e., use `stock-zlib` then).

Thus I think this block has to be removed.

Reply:
I've made it non-fatal, this way a warning gets printed if a common target would fail that _actually_ uses the intrinsics since the cross versions are too old to support it.